nearlib/dev.js

```diff
@@ -26,23 +27,23 @@ module.exports = {
             options.accountId = devAccountName;
             options.key = devKey;
         }
-        const keyStore = (options.deps && options.deps.keyStore) || new nearlib.BrowserLocalStorageKeystore();
-        const nodeUrl = options.nodeUrl || (await this.getConfig()).nodeUrl || 'http://localhost:3030';
+        this.deps = options.deps || {};
+        this.deps.keyStore = this.deps.keyStore || new nearlib.BrowserLocalStorageKeystore();
+        this.deps.storage = this.deps.storage || window.localStorage;
+        const nodeUrl = options.nodeUrl || (await this.getConfig()).nodeUrl || localNodeUrl;
         const nearClient = new nearlib.NearClient(
-            new nearlib.SimpleKeyStoreSigner(keyStore), new nearlib.LocalNodeConnection(nodeUrl));
+            new nearlib.SimpleKeyStoreSigner(this.deps.keyStore), new nearlib.LocalNodeConnection(nodeUrl));
         this.near = new nearlib.Near(nearClient);
         if (options.accountId) {
-            keyStore.setKey(options.accountId, options.key);
+            this.deps.keyStore.setKey(options.accountId, options.key);
         } else {
-            await this.getOrCreateDevUser(options.deps);
+            await this.getOrCreateDevUser();
         }
         return this.near;
     },
-    getOrCreateDevUser: async function (deps = {}) {
-        const storage = deps.storage || window.localStorage;
-        let tempUserAccountId = storage.getItem(storageAccountIdKey);
-        const localKeyStore = deps.keyStore || new nearlib.BrowserLocalStorageKeystore();
-        const accountKey = await localKeyStore.getKey(tempUserAccountId);
+    getOrCreateDevUser: async function () {
```

Comment:
I want to remove this from usage from nearstudio. There is no reason for near studio to use this since Bowen added code to create account for contract deployment

Reply:
hmm, I didn't even realize it's used there, I thought it only used by fiddles